src/views/tiles/ChapterView.js

```diff
@@ -0,0 +1,64 @@
+import ui.ImageView as ImageView;
+
+import ui.TextView as TextView;
+import ui.ScoreView as ScoreView;
+
+exports = Class(ImageView, function (supr) {
+	this.init = function (opts) {
+		supr(this, 'init', [opts]);
+
+		this._adventureMapView = opts.adventureMapView;
+		this._chapterSettings = opts.chapterSettings;
+		this._characterSettings = opts.chapterSettings.characterSettings || {};
+		this._titleSettings = this._characterSettings.title;
+		this._idSettings = this._characterSettings.id;
+
+		this._titleText = new TextView(merge({
+			superview: this,
+		}, this._titleSettings));
+
+		this._idText = new ScoreView(merge({
+			superview: this
+		}, this._idSettings));
```

Comment:
why do we have a textview and a score view ?

Reply:
score view to show chapter nos, text view for chapter title can be localized as well.

Reply:
okay understood